lib/svgo.js

```diff
@@ -20,7 +20,7 @@ var SVGO = module.exports = function(config) {
     this.config = CONFIG(config);
 };
 
-SVGO.prototype.optimize = function(svgstr) {
+SVGO.prototype.optimize = function(svgstr, info) {
```

Comment:
good point, things like floatPrecision too, should be global more easily instead of spread like now https://github.com/svg/svgo/blob/master/lib/svgo/config.js#L36 I'll see if it could be done with your way